pyproject.toml

```diff
@@ -22,7 +22,8 @@ dependencies = [
   "photutils >=1.13.0",
   "pyparsing >=2.4.7",
   "requests >=2.26",
-  "roman_datamodels>=0.22.0",
+  "romancal @ git+https://github.com/spacetelescope/romancal.git",
```

Comment:
Let's add this only as a test dependency?  Something like jdaviz is also better as an optional dependency; we don't want the SDP pipeline to require that.  Also tox feels better as a test dependency.  I don't have a sense for pip.